<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelo349</class>
 <widget class="QDialog" name="modelo349">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modelo 349</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>NIF:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="niflineEdit">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>EJERCICIO:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ejerciciocomboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Período:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="periodocomboBox">
         <item>
          <property name="text">
           <string>1T Primer trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2T Segundo trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3T Tercer trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4T Cuarto trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0A Periodicidad anual</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01 Enero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02 Febrero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03 Marzo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04 Abril</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05 Mayo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06 Junio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07 Julio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08 Agosto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09 Septiembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 Octubre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11 Noviembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 Diciembre</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Apellidos y nombre o razón social:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nombrelineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="fechagroupBox_2">
       <property name="title">
        <string>Selección de fechas</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>INICIAL:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="inicialdateEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FINAL:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="finaldateEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="complgroupBox">
       <property name="title">
        <string>Complementaria / Sustitutiva</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QRadioButton" name="sustitutivaradioButton">
          <property name="text">
           <string>Sustitutiva</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="complementariaradioButton">
          <property name="text">
           <string>Complementaria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Núm.declaración anterior:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="anteriorlineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Persona con quien relacionarse</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Teléfono:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="telefonolineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Apellidos y nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nombrerellineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="gen349pushButton">
       <property name="text">
        <string>&amp;Genera 349</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/pad.png</normaloff>:/iconos/graf/pad.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cargapushButton">
       <property name="text">
        <string>&amp;Cargar datos</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/refresh.xpm</normaloff>:/iconos/graf/refresh.xpm</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrarpushButton">
       <property name="text">
        <string>&amp;Borrar línea</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="datospushButton">
       <property name="text">
        <string>&amp;Datos cuenta</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/text-x-vcard.png</normaloff>:/iconos/graf/text-x-vcard.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mayorpushButton">
       <property name="text">
        <string>&amp;Mayor</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/mayorg.xpm</normaloff>:/iconos/graf/mayorg.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton">
       <property name="text">
        <string>C&amp;opiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimepushButton">
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="latexpushButton">
       <property name="text">
        <string>&amp;Editar Latex</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/tex.png</normaloff>:/iconos/graf/tex.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>modelo349</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>859</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>898</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
